ocaml/idl/datamodel_vm.ml

```diff
@@ -1033,7 +1033,6 @@ let pool_migrate =
       ; Api_errors.other_operation_in_progress
       ; Api_errors.vm_is_template
       ; Api_errors.operation_not_allowed
-      ; Api_errors.vm_migrate_failed
```

Comment:
Why is this removed? Are you sure this error is not raised by this call?

Reply:
I copied the errors from `vm.pool_migrate` for `vm.restart_device_models`, yes, I checked the code, it is not raised from the `vm.pool_migrate` API call.